packages/sdk-core/src/model/report/BacktraceErrorType.ts

```diff
@@ -0,0 +1 @@
+export type BacktraceErrorType = 'Message' | 'Exception' | 'Uhnandled exception' | 'OOMException' | 'Hang';
```

Comment:
if we're having `Unhandled exception` with a whitespace, maybe we should also have `OOMException`?

Reply:
@perf2711 we follow everything that we have in our libraries - I believe we can make an excuse for OOMException but let's not do that now. I will need to verify all our libraries to verify that